### ISSUE TITLE
Add missing logo image for Docusaurus

### ISSUE THREAD

**Issue by dtinth** [fresh-app/fresh-app.github.io#4]
## Background

Right now the website is missing an image for Docusaurus.

![image](https://user-images.githubusercontent.com/193136/193980029-6d4dbae1-1c26-4e1d-a184-58b3ddebccc4.png)

## Task

- Add a Docusaurus image and make it show on the home page.
  - The image files should be put in `.vitepress/icons`.
  - The logic to select which icon to display should be updated. The logic lives in `.vitepress/icons/index.ts`.
  - SVG file preferred, if use PNG, image size must be at least 64x64.
- Your PR should change only the files in `.vitepress/icons`.
- **Please include a screenshot in your PR to confirm that it is working.**

## How to develop this project

The README file is currently missing. Current steps to develop this project is as follows:

- Install deps
   ```
   yarn
   ```
- Load latest build data
   ```
   yarn tsx scripts/load.ts
   ```
- Generate page files
   ```
   yarn tsx scripts/generate.ts
   ```
- Start development server
   ```
   yarn dev
   ```

## Hacktoberfest

If you would like to work on this task, please write a comment stating your intent. We will then assign the issue to you. To ensure continuity, the issue will be unassigned after 3 days of inactivity — so please keep us updated.

**Getting assigned to an issue is NOT a reservation.** It is simply to let others know that someone is already working on it to avoid duplicated efforts. The first person who submits a valid Pull Request (PR) that resolves the issue will have their PR merged, regardless of who was assigned to the issue. However, if you are assigned to an issue and start working on it, but someone else submits a valid PR before you, you can still submit a valid PR even if it has conflicts. The PR will be given the "hacktoberfest-accepted" label and the issue will be closed.

If you encounter any problem, please __include relevant logs and screenshot__ so it’s easy to troubleshoot.

**Comment by Ishika-jain** [comment]
Can you assign this to be me?

**Comment by dtinth** [comment]
@Ishika-jain Please go ahead, thanks!

**Comment by ojninja16** [comment]
hi @dtinth ...I would also like to work on it.....

**Comment by dtinth** [comment]
@ojninja16 This issue is already assigned to @Ishika-jain but I did not see a fork yet. There might be overlapping effort but if you are ready to work on it right away please feel free.

I have clarified what I mean by “assigned:”

> **Getting assigned to an issue is NOT a reservation.** It is simply to let others know that someone is already working on it to avoid duplicated efforts. The first person who submits a valid Pull Request (PR) that resolves the issue will have their PR merged, regardless of who was assigned to the issue. However, if you are assigned to an issue and start working on it, but someone else submits a valid PR before you, you can still submit a valid PR even if it has conflicts. The PR will be given the "hacktoberfest-accepted" label and the issue will be closed.

So, I will merge the 1st valid PR that is submitted for this issue.

**Comment by Savio629** [comment]
After setting up the project on my local system, there's a error popping up in index.js file that "cannot found the module or corresponding type declarations" for svg and png files even though I didn't change any of the code. Is there something I haven't install? If you got any idea can you ping me with a solutions.... @dtinth

**Comment by dtinth** [comment]
@Savio629 Please include the logs and screenshot.

**Comment by Savio629** [comment]
What kind of log files? 
I have attached the index.ts file ⬇
<img src="https://user-images.githubusercontent.com/91362589/194265105-3b81c8b8-5913-48e4-926e-3b43c66b51b9.png" width="600">
Also I have attached package.json file ⬇
<img src="https://user-images.githubusercontent.com/91362589/194265114-c67f7cc3-5578-4e2b-aeb6-02833ccf672e.png" width="600">

**Comment by dtinth** [comment]
@Savio629 Thanks for the screenshot, now I get what you mean.

These errors is only diagnostic output in VS Code. It will not affect the build (which is a separate process). This means you can still run the development server even if there are errors in VS Code. I have not set up proper tooling in VS Code yet.

So, you can ignore these errors as long as the site is working. I will create a separate issue for fixing this. Thanks!

**Comment by Savio629** [comment]
@dtinth But the thing is it isn't visible when I run the server.
Should I use some different IDE to make changes ?
Because I won't be able to check if the logo is working properly or not...
<img src="https://user-images.githubusercontent.com/91362589/194350913-51837ae0-d882-4164-a13a-9c9eb9a7f6ac.png" width="600">

**Comment by Savio629** [comment]
I checked the issue you created regarding ts files in vscode and soon as you accept the changes related to it, I'll start working on the logo issue...

**Comment by dtinth** [comment]
@Savio629 Please make sure you run yarn load and yarn generate before starting the dev server. For more detailed instructions please check out the README file.
